src/main/java/com/code_intelligence/jazzer/driver/Driver.java

```diff
@@ -140,6 +138,9 @@ public static int start(List<String> args, boolean spawnsSubprocesses) throws IO
       }
     }
 
+    // Installing the agent after "findFuzzTarget" leads to an asan error in it on
+    // "Class.forName(targetClassName)", but only during native fuzzing.
+    AgentInstaller.install(Opt.hooks);
```

Comment:
@fmeum Any idea why the agent needs to be installed before `findFuzzTarget` is called?

Reply:
We have a hook on `Class.forName`, but that's only installed by the agent... I have to take a deeper look, I don't know.

Reply:
@bertschneider Could you create a ticket for this and assign it to me so that we don't lose track of it?